frontend/ui/elements/screensaver_menu.lua

```diff
@@ -153,12 +154,19 @@ return {
                         end,
                     },
                     {
-                        text = _("Stretch covers and images to fit screen"),
+                        text_func = function()
+                            return T(_("Stretch covers and images to fit screen: %1%"),
+                                G_reader_settings:readSetting("screensaver_stretch_limit_percentage",8))
```

Comment:
One line's also fine by me (but space after comma please ^_^).
```suggestion
                            return T(_("Stretch covers and images to fit screen: %1%"),
                                G_reader_settings:readSetting("screensaver_stretch_limit_percentage", 8))
```